packages/asset/hardhat.config.ts

```diff
@@ -20,5 +20,59 @@ const config: HardhatUserConfig = {
       },
     ],
   },
+  namedAccounts: {
+    deployer: {
+      default: 0,
+    },
+    sandAdmin: {
+      default: 0, // TODO: make same as core
+    },
+    filterOperatorSubscription: 'deployer',
+    upgradeAdmin: 'sandAdmin',
+    catalystRoyaltyRecipient: '0xB37d8F5d1fEab932f99b2dC8ABda5F413043400B', // testing wallet
+    trustedForwarder: '0xf5D0aDF879b717baA5c444B23D7Df0D5e3e3cBD0', // fake
+    assetAdmin: 'sandAdmin',
+    assetCreateAdmin: 'sandAdmin',
+    assetRevealAdmin: 'sandAdmin',
+    catalystMinter: 'sandAdmin',
+    catalystAdmin: 'sandAdmin',
+    tsbAssetMinter: 'sandAdmin', // For Special Minting of TSB Exclusive assets only
+    authValidatorAdmin: 'sandAdmin',
+    uriSetter: 'sandAdmin',
+    backendAuthWallet: {
+      default: 2,
+    },
+  },
+  defaultNetwork: 'hardhat',
+  networks: {
+    hardhat: {
+      forking: {
+        enabled: false, // note: if set to true then CI will fail
+        blockNumber: 16000000,
+        url: process.env.ETH_NODE_URI_POLYGON || 'http://localhost:8545',
+      },
+      loggingEnabled: false,
+      chainId: 1337,
+      allowUnlimitedContractSize: false,
+      mining: {
+        auto: true,
+        interval: 1000,
+        mempool: {
+          order: 'fifo',
+        },
+      },
+    },
+    'polygon-mumbai': {
+      url: 'https://rpc-mumbai.maticvigil.com',
+      accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : undefined,
+      chainId: 80001,
+      verify: {
+        etherscan: {
+          apiKey: process.env.ETHERSCAN_API_KEY,
+          apiUrl: 'https://api-mumbai.polygonscan.com/',
+        },
+      },
+    },
+  },
```

Comment:
The above section should no longer be needed and should be removed from hardhat config

Reply:
Actually I don't think we should have any changes to the asset hardhat config in this PR.